rasa/data.py

```diff
@@ -95,7 +95,7 @@ def get_core_nlu_files(
             story_files.update(new_story_files)
             nlu_data_files.update(new_nlu_data_files)
 
-    return story_files, nlu_data_files
+    return sorted(story_files), sorted(nlu_data_files)
```

Comment:
ideally that would also be covered by a test

Reply:
honestly writing tests is not my forte 😅 there's no test for this function at all from what i can see. do you mind taking over?

Reply:
actually, due to our wonderful testing file structure, i just didn't see the right tests. It's actulally implicitly covered by https://github.com/RasaHQ/rasa/blob/1.8.x/tests/core/test_data.py#L157 imo. 
I need to fix that test anyways